src/dashboards/Compact/index.tsx

```diff
@@ -100,6 +115,62 @@ const EnturDashboard = ({ history }: Props): JSX.Element => {
 
     const maxWidthCols = COLS[breakpoint]
 
+    const prevNumberOfStopPlaces = usePrevious(numberOfStopPlaces)
+
+    const [modalVisible, setModalVisible] = React.useState(false)
```

Comment:
Kan godt kun bruke `useState` i staden for `React.useState`, sidan `useState` er importert øverst.